kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -616,6 +623,201 @@ def _short_info(cterm: CTerm) -> Iterable[str]:
+                                KApply(
+                                    '#buf(_,_)_BUF-SYNTAX_Bytes_Int_Int',
+                                    KVariable('_A7'),
+                                    KVariable('_A8'),
+                                ),
+                            ],
+                        ),
+                        KVariable('_A9'),
+                    ],
+                ),
+                KApply(
+                    '#return___EVM_KItem_Int_Int',
+                    [
+                        KVariable('_A8'),
+                        KVariable('_A9'),
+                    ],
+                ),
+                KApply('#pc[_]_EVM_InternalOp_OpCode', [KApply('STATICCALL_EVM_CallSixOp')]),
+                KApply('#execute_EVM_KItem'),
+                KVariable('_A12'),
+            ]
+        ),
+        KSequence(
+            [
+                KApply(
+                    call_code,
+                    call_var,
+                ),
+                KApply(
+                    '#return___EVM_KItem_Int_Int',
+                    [
+                        KVariable('_A8'),
+                        KVariable('_A9'),
+                    ],
+                ),
+                KApply('#pc[_]_EVM_InternalOp_OpCode', [KApply('STATICCALL_EVM_CallSixOp')]),
+                KApply('#execute_EVM_KItem'),
+                KVariable('_A10'),
+            ]
+        ),
+    ]
+
+    def _cons_intersection(constraints: Iterable[KInner]) -> KInner:
+        args_filtered = [apply.args for apply in constraints if isinstance(apply, KApply)]
+        bool_filtered = [arg for args in args_filtered for arg in args if arg != TRUE]
+        return andBool(bool_filtered)
+
+    def _get_raw_constraint(kast: KInner) -> KInner:
+        # if type(kast) is KApply:
+        #     print(kast.label.params)
+        #     if kast.label.params == (KSort(''), GENERATED_TOP_CELL) and kast.args[0] == TRUE:
+        #         return kast.args[1]
+        # return kast
+        if type(kast) is KApply:
+            return kast.args[1]
+        return kast
+
+    for name, contract in foundry.contracts.items():
+        if len(contracts) == 0 or name in contracts:
+            for test in contract.tests:
+                if len(set(tests)) == 0 or _test_name(name, test) in tests:
+                    proof_digest = foundry.proof_digest(name, test)
+                    if not APRProof.proof_exists(proof_digest, apr_proofs_dir):
+                        _LOGGER.warning(f'Proof does not exists: {_test_name(name, test)}')
+                    else:
+                        apr_proof = APRProof.read_proof(proof_digest, apr_proofs_dir)
+                        if apr_proof.status is not ProofStatus.PASSED:
+                            _LOGGER.warning(f'Koverage not running on not passed proof: {_test_name(name, test)}')
+                        else:
+                            kcfg = apr_proof.kcfg
+
```

Comment:
This intermediate is only used in 4 places as far as I can tell. I would rename `apr_proof` to just `proof`, and then just use `proof.kcfg` everywhere instead of the intermediate.